docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -50,7 +52,7 @@ You can create a cluster in the free tier of [YugabyteDB Managed](../../../quick
 
 ### Transaction priority between concurrent operations
 
-Let's create an example scenario of an accounts table, and insert a row into as follows:
+Consider an example scenario of an maintaining a bank account. Create the accounts table, and insert a row into as follows:
```

Comment:
Consider an example scenario of maintaining a bank account. Create the accounts table and insert a row into it, as follows: